rules/sha/index.bzl

```diff
@@ -4,7 +4,7 @@ def sha(name, srcs, **kwargs):
         name = name,
         srcs = srcs,
         outs = [name + ".sum"],
-        cmd = "echo \"$(SRCS)\" | xargs find | sort | xargs shasum | shasum | awk '{ print $$1 } > $@",
+        cmd = "xargs find $(SRCS) -type f | sort | xargs shasum | shasum | awk '{ print $$1 }' > $@",
```

Comment:
Instead of `xargs find` I think just `find` works

Reply:
Whoops, leftover from my pipe-ing days. Fixed.